Library/Formula/thruk.rb

```diff
@@ -0,0 +1,105 @@
+class Thruk < Formula
+  desc "Monitoring web interface with support for multiple backends."
+  homepage "http://www.thruk.org/"
+  url "https://github.com/sni/Thruk/archive/v2.06.tar.gz"
+  sha256 "85cd8d44a296887ab3245bfb2d988f4dfe4922f6d85848b05d6d5a5867a8c3d1"
+
+  depends_on "autoconf"   => :build
+  depends_on "automake"   => :build
+  depends_on "cpanminus"      => :build
+  depends_on "libtool"    => :build
+  depends_on "nagios"     => :recommended
+  depends_on "pkg-config" => :build
+
+  patch :DATA
+
+  private
+
+  def perl_modules
+    (%w[Plack-1.tar.gz Plack::Util FCGI GD-2.44.tar.gz Template Date::Manip] +
+     %w[Excel::Template Log::Log4perl-1.34.tar.gz Log::Dispatch::File MIME::Lite] +
+     %w[LWP::Protocol::connect DBD::mysql HTML::Escape HTML::Lint File::BOM Test::Cmd] +
+     %w[Test::Perl::Critic Test::Pod::Coverage Perl::Critic::Policy::Dynamic::NoIndirect] +
+     %w[Perl::Critic::Policy::NamingConventions::ProhibitMixedCaseSubs] +
+     %w[Perl::Critic::Policy::ValuesAndExpressions::ProhibitAccessOfPrivateData] +
+     %w[Perl::Critic::Policy::Modules::ProhibitPOSIXimport WWW::Mechanize::Firefox] +
+     %w[Test::JavaScript Test::Vars Devel::Cycle]).join(" ").freeze
+  end
+
+  public
+
+  def install
+    system "cpanm", "--quiet", "--installdeps", "--self-contained", perl_modules
+    system "./configure", "--prefix=#{prefix}",
+                          "--with-logdir=#{prefix}/var/log"
+    system "perl", "Makefile.PL"
+    system "make", "install"
+  end
+
+  test do
+    system "#{bin}/nagexp", "--version"
+    system "#{bin}/naglint", "--version"
+    system "#{bin}/thruk", "--version"
+  end
```

Comment:
Can’t we write a test that does something more substantial than e.g. `--version` or `--help`?